qutebrowser/browser/webkit/webpage.py

```diff
@@ -192,6 +192,18 @@ def _handle_errorpage(self, info, errpage):
             errpage.encoding = 'utf-8'
             return True
 
+    def chooseFile(self, parent_frame: QWebFrame, suggested_file: str) -> str:
+        """Override chooseFiles to (optionally) invoke custom file uploader."""
```

Comment:
Nitpick:

```suggestion
        """Override chooseFile to (optionally) invoke custom file uploader."""
```

(or drop "Override chooseFile to" entirely, both for QtWebEngine above and here)